src/azure-cli/azure/cli/command_modules/redis/tests/latest/test_redis_scenario.py

```diff
@@ -86,11 +86,33 @@ def test_redis_cache_with_tlsversion_and_settings(self, resource_group):
             self.check('tenantSettings.hello', '1')
         ])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_redis')
+    def test_redis_cache_with_redisversion(self, resource_group):
+
+        self.kwargs = {
+            'rg': resource_group,
+            'name': self.create_random_name(prefix=name_prefix, length=24),
+            'location': location,
+            'sku': basic_sku,
+            'size': basic_size,
+            'redis_version': '6'
+        }
+
+        self.cmd('az redis create -n {name} -g {rg} -l {location} --sku {sku} --vm-size {size} --redis-version {redis_version}')
```

Comment:
Can you add tests for updating redis version too?

Reply:
Passing `None` will not cause any updates as it not valid. Update will happen only if valid value is passed which is currently 4 or 6.

Reply:
Then can you just add tests for updating. Like to version 4

Reply:
updating redis version work is in progress and currently it doesn't change the resource redis version, so didn't include update test case